src/app/faq/faq.component.scss

```diff
@@ -0,0 +1,9 @@
+.description {
```

Comment:
**note:** Not good practice to have repeated classes in scss files. Better to set global styles and use ng styles